ext/ManoptLineSearchesExt.jl

```diff
@@ -59,6 +59,7 @@ function (cs::Manopt.LineSearchesStepsize)(
         return α
     catch ex
         if isa(ex, LineSearches.LineSearchException)
+            println(ex)
```

Comment:
I think I would prefer a re-throw of the error here; and maybe we should introduce such errors in our line searches as well.

Reply:
OK, then the easiest solution is to just not catch the error. We could have that in our line searches too.

Reply:
Yes.